flyteplugins/go/tasks/pluginmachinery/tasklog/template.go

```diff
@@ -194,31 +200,53 @@ func (input Input) templateVarsForScheme(scheme TemplateScheme) TemplateVars {
 	return vars
 }
 
+func getDynamicLogLinkTypes(taskTemplate *core.TaskTemplate) []string {
+	if taskTemplate == nil {
+		return nil
+	}
+	config := taskTemplate.GetConfig()
+	if config == nil {
+		return nil
+	}
+	linkType := config["link_type"]
+	if linkType == "" {
+		return nil
+	}
+	dynamicLogLinkTypes := []string{}
+	for _, linkType := range strings.Split(linkType, ",") {
+		// NB: we are going to grandfather in flyin.
+		if linkType == "vscode" {
+			dynamicLogLinkTypes = append(dynamicLogLinkTypes, "flyin")
```

Comment:
Why special case flyin? is `vscode` not enough?

Reply:
Setting the link type to `vscode` is a hack. We should standardize on the name of the plugins (including `flyin`).

Reply:
Can we get rid of vscode then?

Reply:
`vscode` is already being used in [flyin](https://github.com/flyteorg/flytekit/blob/master/plugins/flytekit-flyin/flytekitplugins/flyin/vscode_lib/decorator.py#L487). We can make the change in flytekit (and I think we should), but this is going to break existing uses.

Reply:
@eapolinario do not use `flyin` as LinkedIn prefers FlyteInteractive

Reply:
I modified all uses of `flyin` to `FlyteInteractive`.